src/core/zowe/core_for_zowe_sdk/logger.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+class Log:
+    """root logger setup and a funtion to customize logger level"""
+
+    dirname = os.path.join(os.path.expanduser("~"), ".zowe/logs")
```

Comment:
The default location of the `.zowe` folder is `~/.zowe` as used here. In Zowe CLI and Zowe Explorer, it can be overridden with the `ZOWE_CLI_HOME` environment variable to point to another directory.

We may want to check for the existence of `os.environ["ZOWE_CLI_HOME"]`, and if it is defined then store the `logs` directory inside it. This could be addressed in a separate issue 🙂

Reply:
I think it could be addressed in a separate issue, too.